x-pack/test/functional/page_objects/uptime_page.ts

```diff
@@ -35,12 +35,15 @@ export function UptimePageProvider({ getPageObjects, getService }: FtrProviderCo
       datePickerStartValue: string,
       datePickerEndValue: string,
       monitorId: string,
-      monitorName: string
+      monitorName?: string
     ) {
       await pageObjects.common.navigateToApp('uptime');
       await pageObjects.timePicker.setAbsoluteRange(datePickerStartValue, datePickerEndValue);
       await uptimeService.navigateToMonitorWithId(monitorId);
-      if ((await uptimeService.getMonitorNameDisplayedOnPageTitle()) !== monitorName) {
+      if (
+        !!monitorName &&
```

Comment:
Do we need `!!` in javascript? It's useful when you want to return a bool, but isn't it equivalent as just omitting it in a boolean expression?

Reply:
Deleted in https://github.com/elastic/kibana/pull/55915/commits/20564716a7c76c32f4814482149afc4c9246a429.